saleor/graphql/shop/types.py

```diff
@@ -63,6 +63,19 @@ class ExternalAuthentication(graphene.ObjectType):
     name = graphene.String(description="Name of external authentication plugin.")
 
 
+class Limits(graphene.ObjectType):
+    channels = graphene.Int()
+    orders = graphene.Int()
+    product_variants = graphene.Int()
+    staff_users = graphene.Int()
+    warehouses = graphene.Int()
```

Comment:
Can these values be null or will they always be numeric?

Reply:
Can be `null`, `null` would mean unlimited in our case

Reply:
👍